OpenRA.Game/Network/OrderManager.cs

```diff
@@ -105,7 +110,7 @@ void CacheChatLine(Color color, string name, string text)
 		public void TickImmediate()
 		{
 			var immediateOrders = localOrders.Where(o => o.IsImmediate).ToList();
-			if (immediateOrders.Count != 0)
+			if (immediateOrders.Count != 0 && GameSaveLastFrame < NetFrameNumber + FramesAhead)
```

Comment:
NetFrameNumber + FramesAhead is used in several places, could be made to its own property

Reply:
Can you suggest a name for this?